Sources/BraintreeVenmo/BTVenmoClient.swift

```diff
@@ -43,8 +43,8 @@ import BraintreeCore
     /// In the Venmo flow this will be the payment context ID
     private var payPalContextID: String?
 
-    /// Used for sending the type of flow, universal vs deeplink to FPTI
-    private var linkType: LinkType?
+    /// Used for sending the type of flow, universal, to FPTI
+    private var linkType: LinkType = .universal
```

Comment:
Is linkType always universal and if so do we still need this top level property?

Reply:
It will always be universal

> do we still need this top level property?
`linkType` is used throughout `BTVenmoClient` so we'll still need the global property for sure. Where else do you suggest to move it?

Reply:
Good call, I've made the changes!